.circleci/deploy.sh

```diff
@@ -1,45 +1,43 @@
 #!/bin/bash -e
 
 MODULE=$1
-DIR=$(dirname "$0")
-
-if grep -e "$MODULE" -e 'all' "$DIR/../packages/test.list" > /dev/null; then
```

Comment:
no need to grep `test.list` here anymore. we're going through all the packages anyway